src/ui/simulator/application/study.cpp

```diff
@@ -573,7 +573,8 @@ void MarkTheStudyAsModified(const Data::Study::Ptr& study)
     if (!(!study) and study == GetCurrentStudy())
         MarkTheStudyAsModified();
 }
-
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wvolatile"
```

Comment:
You could place these around the code that triggers the warning to make it clear what piece of code triggers the warning